lib/core-integration/src/Test/Integration/Scenario/API/Shared/Transactions.hs

```diff
@@ -0,0 +1,226 @@
+                        { "cosigner#0": #{accXPubDerived} },
+                      "template":
+                          { "all":
+                             [ "cosigner#0",
+                               "cosigner#1",
+                               { "active_from": 120 }
+                             ]
+                          }
+                    }
+                } |]
+        rPost <- postSharedWallet ctx Default payload
+        verify (fmap (swapEither . view #wallet) <$> rPost)
+            [ expectResponseCode HTTP.status201
+            ]
+
+        let (ApiSharedWallet (Left wal)) = getFromResponse id rPost
+
+        let metadata = Json [json|{ "metadata": { "1": { "string": "hello" } } }|]
+
+        rTx <- request @(ApiConstructTransaction n) ctx
+            (Link.createUnsignedTransaction @'Shared wal) Default metadata
+        verify rTx
+            [ expectResponseCode HTTP.status403
+            , expectErrorMessage errMsg403SharedWalletPending
+            ]
+
+    it "SHARED_TRANSACTIONS_CREATE_01 - Can create tx for an active shared wallet" $ \ctx -> runResourceT $ do
+        m15txt <- liftIO $ genMnemonics M15
+        m12txt <- liftIO $ genMnemonics M12
+        let (Right m15) = mkSomeMnemonic @'[ 15 ] m15txt
+        let (Right m12) = mkSomeMnemonic @'[ 12 ] m12txt
+        let passphrase = Passphrase $ BA.convert $ T.encodeUtf8 fixturePassphrase
+        let index = 30
+        let accXPubDerived = accPubKeyFromMnemonics m15 (Just m12) index passphrase
+        let payload = Json [json| {
+                "name": "Shared Wallet",
+                "mnemonic_sentence": #{m15txt},
+                "mnemonic_second_factor": #{m12txt},
+                "passphrase": #{fixturePassphrase},
+                "account_index": "30H",
+                "payment_script_template":
+                    { "cosigners":
+                        { "cosigner#0": #{accXPubDerived} },
+                      "template":
+                          { "all":
+                             [ "cosigner#0" ]
+                          }
+                    }
+                } |]
+        rPost <- postSharedWallet ctx Default payload
+        verify (fmap (swapEither . view #wallet) <$> rPost)
+            [ expectResponseCode HTTP.status201
+            ]
+
+        let walShared@(ApiSharedWallet (Right wal)) = getFromResponse id rPost
+
+        let metadata = Json [json|{ "metadata": { "1": { "string": "hello" } } }|]
+
+        rTx1 <- request @(ApiConstructTransaction n) ctx
+            (Link.createUnsignedTransaction @'Shared wal) Default metadata
+        verify rTx1
+            [ expectResponseCode HTTP.status403
+            , expectErrorMessage errMsg403EmptyUTxO
+            ]
+
+        let amt = 10 * minUTxOValue (_mainEra ctx)
+        fundSharedWallet ctx amt walShared
+
+        rTx2 <- request @(ApiConstructTransaction n) ctx
+            (Link.createUnsignedTransaction @'Shared wal) Default metadata
```

Comment:
It would be nice actually to demonstrate in the integration tests all that we expect to work is working, i.e. constructing:
 - single output tx
 - multi ouput tx
 - multi assets tx
 - metadata
 - use of validity intervals

Reply:
sounds reasonable. I have added testing for all cases except multi assets tx as this topic will require separate treatment and ticket. I also added `fixtureSharedWallet` as you requested in early comment